mocha/main.spec.js

```diff
@@ -1,91 +1,90 @@
-import moch from "mocha";
-import axios from "axios";
-import fs from "fs";
-
-import chai from "chai";
+import axios from 'axios'
+import fs from 'fs'
+import chai from 'chai'
+import dotenv from 'dotenv'
 
 /**
  * TODO: Probably we need ckanClient and frictionless to be passed as an arguments in this test suite.
  * It's unclear what version should we test against? That's why I decided to install them in dev deps.
  * This needs to be clarified
  *  */
```

Comment:
@cuducos and @cotts once we come up with how to use those packages (`peerDependencies` vs providing as a params), we can remove this comment too